batch_job/src/work_queue_factory.c

```diff
@@ -1316,10 +1316,10 @@ int main(int argc, char *argv[])
 	
 	
 	if(runos_os) {
-		cmd = string_format("cp \"$(which vc3-builder)\" '%s'", scratch_dir);
+		cmd = string_format("cp '/afs/crc.nd.edu/group/ccl/software/vc3-builder' '%s'", scratch_dir);
```

Comment:
Please update to 
/afs/crc.nd.edu/group/ccl/software/vc3-builder-src/vc3-builder

I already created that location.

Reply:
Also use a #define with the path, as with runos.